Leetcode/EigenDemo/eigenPractice.hpp

```diff
@@ -0,0 +1,28 @@
+//
+//  eigenPractice.hpp
+//  EigenDemo
+//
+//  Created by dsh on 2021/7/18.
+//
+
+#ifndef eigenPractice_hpp
+#define eigenPractice_hpp
+
+#include <stdio.h>
+#include <iostream>
+
+#include "Eigen/core"
+#include "Eigen/Dense"
+
+using namespace Eigen;
+
+#define MATRIX_SIZE 50
+
+class eigenPractice {
+public:
```

Comment:
```suggestion
  public:
```

https://google.github.io/styleguide/cppguide.html#Class_Format

The public:, protected:, and private: keywords should be indented one space.
但你没有用 2 个空格作为缩进，这里可以进两格。